scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/shared/SchedulerConfig.java

```diff
@@ -260,4 +264,15 @@ public long getTagSuggestionDelay(){
     public int getLivelogsRefreshTime() {
         return Integer.parseInt(properties.get(LIVELOGS_REFRESH_TIME));
     }
+
+    /**
+     * @return the catalog url or an empty string if none has been defined
+     */
+    public String getCatalogUrl() {
```

Comment:
why not letting return null and check if the url is null on this method client? Will clean the code a bit